content/use-cases/tokenization/stablecoin-issuer.md

```diff
@@ -4,504 +4,173 @@ parent: tokenization.html
 
-- The name of the transmittor,
-- The account number of the transmittor, if used,
-- The address of the transmittor,
-- The identity of the transmittor's financial institution,
-- The amount of the transmittal order,
-- The execution date of the transmittal order, and
-- The identity of the recipient's financial institution.
+### Global Freeze
 
-<!-- SPELLING_IGNORE: transmittor -->
+If you see signs of suspicious activity, you can enact a global freeze on your account to prevent users from sending your tokens to each other and trading your token in the decentralized exchange. 
 
-See also:
+![Global Freeze](img/uc-stablecoin-global-freeze.png)
 
-- [Funds “Travel” Regulations: Questions & Answers ](https://www.fincen.gov/resources/statutes-regulations/guidance/funds-travel-regulations-questions-answers)
+See [Enact Global Freeze](enact-global-freeze.html)
 
-### Fee Disclosure and Tracing Funds
 
-- In the United States, Dodd Frank 1073 Electronic Fund Transfer Act (Regulation E) requires banks to provide information on cost and delivery terms for international payments originating in the US including exchange rate, fees, and the amount to be received by the designated recipient in the foreign country. "Pre-payment disclosure" is provided to a consumer when requesting an international electronic payment and “receipt disclosure” is provided to a consumer at the time consumer authorizes the transfer.
+### Clawback
 
-    See also:
+Clawback is an optional setting that you can choose before you begin to distribute your stablecoin. For regulatory purposes, some issuers _must_ have the ability to recover issued tokens after they are distributed to accounts. For example, if an issuer were to discover that tokens were sent to an account sanctioned for illegal activity, the issuer could recover, or _claw back_, the funds.
 
-    - [The Consumer Financial Protection Bureau description of the regulation and extensions for banks](https://www.consumerfinance.gov/rules-policy/final-rules/electronic-fund-transfers-regulation-e/#rule)
+See [Clawback](clawback.html).
 
-- In the European Union, EU Funds Transfer Regulation requires that the originator’s bank, the beneficiary’s bank, and any intermediary banks include certain details of the payer and payee in transaction details to detect, investigate, and prevent money laundering and terrorist financing.
+![Clawback](img/uc-stablecoin-clawback.png)
 
-    See also:
+### Burn
 
-    - [EU Regulation (EC) No 1781/2006 description](http://eur-lex.europa.eu/LexUriServ/LexUriServ.do?uri=OJ:L:2006:345:0001:0009:EN:PDF)
+One way to manage the value of a stablecoin is to destroy, or _burn_ stablecoins. When you burn stablecoins, you reduce the number of tokens in circulation. That will typically increase the value of the remaining tokens as supply falls.
```

Comment:
This goes back to my comment above about stretching the definition of a stablecoin. The value per unit of a stablecoin is supposed to be fixed with respect to its collateral, so burning stablecoins _must_ mean that there is less collateral, unless the coin has become un-pegged and is no longer a stablecoin.